GreedyCounter/Form1.cs

```diff
@@ -111,5 +160,29 @@ private void AppendTextToOutput(string text)
                 textBoxOutput.AppendText($"{text}{Environment.NewLine}");
             }*/
         }
+
+        void timer_Tick(object sender, EventArgs e)
+        {
+
+            //updates the Output text box
+            sb.Clear();
+            sb.AppendLine("Greedy Count:");
+            // Add the keypair to a string via stringbuilder
+            foreach (KeyValuePair<string, int> kvp in cDict)
+            {
+                //sb.AppendFormat("{0}: {1} \n", kvp.Key, kvp.Value.ToString());
+                sb.AppendLine(kvp.Key + ": " + kvp.Value.ToString());
```

Comment:
[nit] I would use string interpolation here. This line could be ```sb.AppendLine($"{kvp.Key} :  {kvp.Value}");``` Which I think looks nicer.

Reply:
Oh nice! I agree